packages/trader/test/RangeTrader.ts

```diff
@@ -0,0 +1,338 @@
+import { web3, assert } from "hardhat";
```

Comment:
These tests test the `RangeTrader` module directly, just like the `Liquidator.js` tests test the liquidator. They verify the range trading behaviour against expected market movements.